clients/slmpclient/slmp_client.go

```diff
@@ -180,3 +182,9 @@ func (c SlmpClient) GetServiceVersions(serviceID string) (models.Versions, error
 func getSlmpURL(org, space string) string {
 	return "slprot/" + org + "/" + space + "/slp"
 }
+
+func getSlmpURLEncoded(org, space string) string {
+	orgEncoded := strings.Replace(url.QueryEscape(org), "+", "%20", -1)
```

Comment:
Same here?

Reply:
Only original function getRestURL is kept